Tasks/SonarQubePostTest/PRCA/PostComments-Server.ps1

```diff
@@ -239,9 +240,8 @@ function GetCodeFlowLatestIterationId
     Assert ($review -ne $null) "Could not retrieve the review"
     Assert (HasElements $review.Iterations) "No iterations found on the review"
     
-    # TODO: is this the best way to find the id ?
```

Comment:
THere is an "UpdateDate" property there, but it might not take into account timezone so actually comparing IDs might be ugly, but rather reliable. Note that this was reviewed by the Code Review team.